app/assets/stylesheets/nfg_ui/network_for_good/email/application.scss

```diff
@@ -0,0 +1,52 @@
+@import "settings/settings";
+@import "foundation-emails";
+
+// @import "util/util";
```

Comment:
@kylejmorin any reason we should keep these commented out pieces at this point?

Reply:
I can't remember why they were commented out besides the fact we didn't use or need them. We can always bring them back with version control. Cool to remove!